src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -19,6 +19,16 @@
 
 "General_BackButtonTitle" = "Zurück";
 
+/* General - Links */
+
+"General_moreInfo_URL" = "https://www.coronawarn.app/de/faq/";
+
+"General_moreInfo_URL_EN5" = "https://www.coronawarn.app/de/faq/#ENError5";
+
+"General_moreInfo_URL_EN11" = "https://www.coronawarn.app/de/faq/#ENError11";
+
+"General_moreInfo_URL_EN13" = "https://www.coronawarn.app/de/faq/#ENError13";
+
```

Comment:
I shifted these around as they are used in several places (not just ExposureSubmission) anymore.

Reply:
Rewrote the Exposure Submission error handlings to use these links, removed the old traces of the links.